lit-base/model_loader.py

```diff
@@ -0,0 +1,50 @@
+import yaml
+
+import mxnet as mx
+
+import tempfile
+from urllib import urlretrieve
+
+
+# load model
+def load_model(s_fname, p_fname):
+    """
+    Load model checkpoint from file.
+    :return: (symbol, arg_params, aux_params)
+    symbol : Symbol
+        The network configuration.
+    arg_params : dict of str to NDArray
+        Model parameter, dict of name to NDArray of net's weights.
+    aux_params : dict of str to NDArray
+        Model parameter, dict of name to NDArray of net's auxiliary states.
+    """
+    symbol = mx.symbol.load(s_fname)
+    save_dict = mx.nd.load(p_fname)
+    arg_params = {}
+    aux_params = {}
+    for key, value in save_dict.items():
+        tp, name = key.split(':', 1)
+        if tp == 'arg':
+            arg_params[name] = value
+        if tp == 'aux':
+            aux_params[name] = value
+    return symbol, arg_params, aux_params
+
+model_url = yaml.load(open("model_url.yaml", "r"))
```

Comment:
please rename `model_url` file to `config`

Reply:
The `yaml` is exactly for `model_url`. I'm very not sure about this renaming.

Reply:
these are configurable parameters that is passed to the function. `model_config` makes sense
I think the point that @kalyc is making is easy you can easily extend this if you want to more configuration parameters like `batch_size`, etc,. 
Also why would you need this yaml file if you use model archive

Reply:
@nswamy @apeforest Modified accordingly